### ISSUE TITLE
Metadata specification for linking to IIIF

### ISSUE THREAD

**Issue by balmas** [distributed-text-services/specifications#162]
Requested by @PietroLiuzzo (and others)

The DTS API should include a recommendation for how to identify links to IIIF Manifests/Images in the metadata.

From @PietroLiuzzo : 
I have looked at what nomisma does, and ended up with something like this,

```
"foaf:depiction" : 
  { "svcs:has_service" : 
    { "doap:implements" : "http://iiif.io/api/image/2/level1.json", 
      "dcterms:conformsTo" : "http://iiif.io/api/image", 
      "@type" : "svcs:Service", 
      "@id" : "https://betamasaheft.eu/api/iiif/ESqdq004/manifest" }, 
   "@type" : "edm:WebResource", 
   "@id" : "https://betamasaheft.eu/manuscript/ESqdq004/viewer" } 
```

Which is not entirely what I really think it should be. I think now that I should have something like this on the navigation level for each pb and use this to point to the correct image instead of pointing to the url of the page where I have the viewer which makes little sense. But how to link then simply the general manifest to the description of the member in the collection?

**Comment by architexte** [comment]
Not easy. It seems to me that a IIIF manifest should be generated according to the `ref`, `start` and `end` query parameters. However, it should be easy to specify a way to declare such a manifest. In the Headers?

**Comment by PonteIneptique** [comment]
Fixed by #177
